jest.config.mjs

```diff
@@ -1,8 +1,14 @@
-module.exports = {
+import yn from "yn";
+
+const collectCoverage = yn(process.env.CI);
```

Comment:
do we really need here a new dev dependency?. Alternate solution `const collectCoverage = process.env.CI ?? false` might work too?

Reply:
I prefer this because I want to properly handle the `CI=false` and `CI=0` cases.